torchmultimodal/modules/layers/quantization.py

```diff
@@ -117,36 +140,34 @@ def _ema_update_embedding(self, encoded_flat: Tensor, codebook_indices: Tensor):
         # Count how often each embedding vector was looked up
```

Comment:
lowercase "count" here?

Reply:
for what reason? this is the start of a new comment